style-guide.md

```diff
@@ -83,9 +83,15 @@ Dla linków, które nie mają polskiego odpowiednika (Stack Overflow, filmy na Y
 
 Użyj tej sekcji, jako spisu tłumaczeń dla często używanych, technicznych pojęć.
 
-| Term   | Translation |
+| Słowo / pojęcie  | Tłumaczenie |
 | ------ | ----------- |
 | Plugin | Wtyczka     |
 | Theme  | Motyw       |
 | Query  | Zapytanie   |
 | String | Łańcuch znaków   |
+| Tutorial | Samouczek lub Poradnik   |
+| Gatsby Docs | Dokumentacja Gatsby (pisane z wielkiej litery)   |
+| Run command | Uruchom komendę   |
+| Development server | Serwer deweloperski   |
```

Comment:
```suggestion
| Development server | Serwer developerski   |
```
nad tym się zastanawiam, bo przez 'w' zawsze kojarzy mi się z tymi od mieszkań i to samo się dostaje po wpisaniu tego w google.

Reply:
> nad tym się zastanawiam, bo przez 'w' zawsze kojarzy mi się z tymi od mieszkań i to samo się dostaje po wpisaniu tego w google.

To prawda, możemy też korzystać ze słowa `programista` lub `deweloper oprogramowania`

Reply:
Ja bym został przy 'w'. Tu kilka przykładów, które znalazłem: 
- https://mensfeld.pl/2011/05/wlasny-serwer-deweloperski-dlaczego-warto/
- https://helpx.adobe.com/pl/dreamweaver/using/set-computer-application-development.html
W polskim języku właściwie, nie ma takiego słowa jak developer.

Reply:
Rozumiem czemu może się tak kojarzyć :) Ale i tak uważam ze skoro takie słowo istnieje w naszym języku i ma sens w tym kontekście, to powinniśmy go użyć.